duckduckgo/iOS
# Voice search settings toggle

Task/Issue URL: https://app.asana.com/0/414709148257752/1202070298493931/f

**Description**:
Adds a setting option to disable/enable voice-search

**Steps to test this PR**:
(on iOS 15)
1. Open the app, check if the voice search is enabled (it should)
2. Open the settings panel, check if the "Hide Voice Search" is disabled (it should)
3. Enable the "Hide Voice Search" option and check that the mic button is not visible on the Omnibar 

(on iOS 14 or lower)
1. Open the settings panel, check if the "Hide Voice Search" cell is hidden. (It should be hidden, since voice-search only works iOS 15+ we should not display this option on older iOS versions)

**OS Testing**:


**Theme Testing**:
